src/transformers/utils/hub.py

```diff
@@ -578,7 +578,7 @@ def download_url(url, proxies=None):
         " that this is not compatible with the caching system (your file will be downloaded at each execution) or"
         " multiple processes (each process will download the file in a different temporary file)."
     )
-    tmp_file = tempfile.mktemp()
+    tmp_file = tempfile.mkstemp()
```

Comment:
I think we need to index here: `mkstemp` returns a tuple, where as `mktemp` returns a single object.

```suggestion
    tmp_file = tempfile.mkstemp()[1]
```